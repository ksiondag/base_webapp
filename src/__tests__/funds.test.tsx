import * as React from "react";
import renderer from 'react-test-renderer';


test('Funds render as a table of names and balances', () => {
    const component = renderer.create(
        <div>
        </div>
    );
});