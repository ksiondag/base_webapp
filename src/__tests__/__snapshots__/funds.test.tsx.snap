// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`funds render as a table of names and balances 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Funds"
      >
        <table
          class="table"
        >
          <thead>
            <tr>
              <th>
                Fund Name
              </th>
              <th>
                Balance
              </th>
              <th>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-default"
                    type="button"
                  >
                    Add
                  </button>
                </div>
              </th>
            </tr>
            <tr>
              <td>
                Woo
              </td>
              <td>
                0
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
            <tr>
              <td>
                Another
              </td>
              <td>
                0
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
            <tr>
              <td>
                Oh really?
              </td>
              <td>
                250
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
            <tr>
              <td>
                Nick
              </td>
              <td>
                100000
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
            <tr>
              <td>
                type a thing
              </td>
              <td>
                0
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
            <tr>
              <td>
                yas
              </td>
              <td>
                250
              </td>
              <td>
                <div
                  class="btn-toolbar"
                  role="toolbar"
                >
                  <button
                    class="btn btn-danger"
                    type="button"
                  >
                    Delete
                  </button>
                </div>
              </td>
            </tr>
          </thead>
        </table>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Funds"
    >
      <table
        class="table"
      >
        <thead>
          <tr>
            <th>
              Fund Name
            </th>
            <th>
              Balance
            </th>
            <th>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-default"
                  type="button"
                >
                  Add
                </button>
              </div>
            </th>
          </tr>
          <tr>
            <td>
              Woo
            </td>
            <td>
              0
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              Another
            </td>
            <td>
              0
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              Oh really?
            </td>
            <td>
              250
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              Nick
            </td>
            <td>
              100000
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              type a thing
            </td>
            <td>
              0
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
          <tr>
            <td>
              yas
            </td>
            <td>
              250
            </td>
            <td>
              <div
                class="btn-toolbar"
                role="toolbar"
              >
                <button
                  class="btn btn-danger"
                  type="button"
                >
                  Delete
                </button>
              </div>
            </td>
          </tr>
        </thead>
      </table>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
